.changeset/public-donkeys-juggle.md

```diff
@@ -0,0 +1,15 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": fix
+---
+Allow edits in arrays and sequences to be concurrent to dependent edits of transactions with violated constraints
```

Comment:
Nit: since changesets are user-facing, and we don't expose "sequences" as a concept in the API, I think you could reduce this to just "...edits in arrays to be..."

Reply:
Don't we expose sequences for root fields?